apps/src/studio/Item.js

```diff
@@ -119,6 +126,12 @@ Item.prototype.update = function () {
   // Draw the item's current location.
   Studio.drawDebugRect("itemCenter", this.x, this.y, 3, 3);
 
+  // In this stationary activity case, we don't need to do any of this
+  // update logic (facing the actor is handled every frame in display())
+  if (this.activity === 'watchActor') {
```

Comment:
Should we expose `watchActor` as an official activity type (permitting it in `setItemActivity`) and adding a new `setToWatch` wrapper API? It seems like a slightly cooler version of activity `none`